airflow/config_templates/config.yml

```
@@ -944,6 +944,14 @@ metrics:
   description: |
     StatsD (https://github.com/etsy/statsd) integration settings.
   options:
+    metrics_use_pattern_match:
+      description: |
+        If true, metrics_allow_list and metrics_block_list will use regex pattern matching
+        anywhere within the metric name instead of only prefix matching at the start of the name.
+      version_added: 2.7.4
```

Since it's a new feature, we need to wait for the next minor release:
```suggestion
      version_added: 2.9.0
```